### PR TITLE
Add event feature for the editing module

### PR DESCRIPTION
**Pull request by ThiefMaster**

### CODE REVIEW

File: indico/modules/events/editing/__init__.py [hunk 1]
```diff
@@ -7,7 +7,27 @@
...
+from indico.util.i18n import _
 
 
 logger = Logger.get('events.editing')
+
+
+class EditingFeature(EventFeature):
+    name = 'editing'
+    friendly_name = _('Paper Editing')
+    description = _('Gives event managers the opportunity to let contributors submit papers to be reviewed and '
```
Comment (meluru) [inline indico/modules/events/editing/__init__.py]: `to be edited` ?

Comment (ThiefMaster) [reply indico/modules/events/editing/__init__.py]: I thought the same, but is "edited" in that context clear?

Comment (ThiefMaster) [reply indico/modules/events/editing/__init__.py]: I thought the same, but is "edited" in that context clear?

Comment (pferreir) [reply indico/modules/events/editing/__init__.py]: I'd choose "edited".

**Pull request merged**